src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version='1.0' encoding='utf-8'?>
+<!DOCTYPE hibernate-configuration PUBLIC "-//Hibernate/Hibernate Configuration DTD 3.0//EN"
+        "http://hibernate.sourceforge.net/hibernate-configuration-3.0.dtd">
+<hibernate-configuration>
+
+    <session-factory>
+        <property name="hibernate.connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="hibernate.connection.url">jdbc:mysql://localhost:3306/test</property>
+        <property name="hibernate.connection.username">root</property>
+        <property name="hibernate.connection.password">ZXCqwe123</property>

```

Comment:
I hope that's not your pass, right?

Reply:
mine)

Reply:
I would strongly recommend not to push your real credentials. Instead, exclude them from the commit or provide some dummy alternatives

Reply:
Still exists.